recipes/qdarkstyle/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "qdarkstyle" %}
+{% set package_name = "QDarkStyle" %}
+{% set version = "2.5.4" %}
+{% set hash_type = "sha256" %}
+{% set hash = "3eb60922b8c4d9cedecb6897ca4c9f8a259d81bdefe5791976ccdf12432de1f0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ package_name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Please change this to:
```
script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
```

Reply:
What's the `-vvv` about? I haven't seen it before.

Reply:
That is basically a 'full' verbose mode of ``pip``
```
-v         Give more output. Option is additive, and can be used up to 3 times.
```